configure.ac

```diff
@@ -647,8 +659,8 @@ AC_SUBST(CXX)
 AC_SUBST(MAINCC)
 AC_MSG_CHECKING(for --with-cxx-main=<compiler>)
 AC_ARG_WITH(cxx_main,
-            AS_HELP_STRING([--with-cxx-main=<compiler>],
-                           [compile main() and link python executable with C++ compiler]),
+            AS_HELP_STRING([--with-cxx-main@<:@=COMPILER@:>@],
+                           [compile main() and link Python executable with C++ compiler specified in COMPILER (default is $CXX)]),
```

Comment:
This is a little misleading: the default is not to link with CXX at all, although the default _value_ when passing `--with-cxx-main` is indeed CXX. (As an aside, I don't think this option is useful anymore; I believe it was only necessary with gcc 2.95, perhaps only on some platforms.)